packages/scoped-custom-element-registry/test/Element.test.html.js

```diff
@@ -104,4 +109,41 @@ describe('Element', () => {
       expect($el.firstElementChild).to.not.be.instanceof(CustomElementClass);
     });
   });
+
+  describe('attributes', () => {
+    it('should call setAttribute', () => {
+      const {tagName, CustomElementClass} = getObservedAttributesTestElement(['foo']);
+      customElements.define(tagName, CustomElementClass);
+      const $el = getHTML('<div></div>');
+      $el.innerHTML = `<${tagName}></${tagName}>`;
```

Comment:
I think this could be simplified to:

```suggestion
      const $el = document.createElement(tagName);
```

here and in the last test.